alert-database/src/main/resources/liquibase/6.4.0/notification-content-processing-column.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="martinch" id="2020-12-01-14-04-07-258">
```

Comment:
The id's should have a unique text describing the change now.  Try something like add_processed_column_to_notification_table

Reply:
I thought we were still using the time as the ID, I gave Martin bad information.

Reply:
We started changing back in September to more descriptive IDs